cmd/broker/filter/main.go

```diff
@@ -124,7 +126,7 @@ func main() {
 		logger.Fatal("Error setting up trace publishing", zap.Error(err))
 	}
 
-	reporter := filter.NewStatsReporter(env.PodName, env.ContainerName)
+	reporter := filter.NewStatsReporter(env.ContainerName, env.PodName+"-"+uuid.New().String())
```

Comment:
To limit length, you might want to use `ChildName`: https://github.com/knative/pkg/blob/master/kmeta/names.go#L39

Reply:
Good to know.